dxr/plugins/xbl/filters.py

```diff
@@ -0,0 +1,23 @@
+from jinja2 import Markup
+from dxr.filters import NameFilterBase, QualifiedNameFilterBase
+from dxr.plugins.xbl.refs import PLUGIN_NAME
+
+
+class _NameFilter(NameFilterBase):
+    lang = PLUGIN_NAME
+
+
+class _QualifiedNameFilter(QualifiedNameFilterBase):
+    lang = PLUGIN_NAME
+
+
+class PropFilter(_QualifiedNameFilter):
+    name = 'prop'
+    is_identifier = True
+    description = Markup('XBL property definition filter: <code>prop:foo</code>')
```

Comment:
Now we hit the cut corner were only the description from the first plugin for a given filter name is shown. So, in a repo that has JS and XBL enabled, people won't be able to discover that XBL props are supported. For this and to see how the various new MXR trees are indexed, it's probably high time to add plugin (language) badging to the Operators and the Switch Tree menu.